app/lib/companies_selector.rb

```diff
@@ -3,7 +3,11 @@
 class CompaniesSelector < ApplicationService
   def call(params)
     relation = Company.all
-    relation = filter_by_condition(relation, params.fetch(:condition)) if params.key?(:condition)
+    relation = if params.key?(:condition)
+                 filter_by_condition(relation, params.fetch(:condition))
+               else
+                 filter_by_condition(relation, 'company_active')
+               end
```

Comment:
```ruby
relation = filter_by_condition(relation, params.fetch(:condition, 'company_active'))
```

wouldn't work the same way?

Reply:
Yes, ok, I can put default value when using `fetch`